scp.py

```diff
@@ -468,3 +468,50 @@ def _set_dirtimes(self):
 class SCPException(Exception):
     """SCP exception class"""
     pass
+
+
+def put(transport, files, remote_path=b'.',
+        recursive=False, preserve_times=False):
+    """
+    Transfer files to remote host.
+
+    @param files: A single path, or a list of paths to be transferred.
```

Comment:
It's tempting to have these just refer to `SCPClient.put`, but I'm guessing the explicitness is helpful for IDEs?